index.bs

```diff
@@ -233,7 +233,7 @@ The [=lock-concept/waiting promise=] is not named in the above code, but is stil
 Further note that if the callback is not `async` and returns a non-promise, the return value is wrapped in a promise that is immediately resolved; the lock will be released in an upcoming microtask, and the [=lock-concept/released promise=] will also resolve in a subsequent microtask.
 </aside>
 
-Each origin has an associated <dfn>held lock set</dfn> which is an [=/ordered set=] of [=lock-concept|locks=].
+Each origin has an associated <dfn>held lock set</dfn> which is a [=set=] of [=lock-concept|locks=].
```

Comment:
No way for you to know this without running bikeshed but: we'll need to use `[=/set=]` to disambiguate the term "set".